spek-ide-plugin/intellij-jvm/src/main/kotlin/org/spekframework/intellij/SpekJvmSettingsEditor.form

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Damn XML! Does the IJ have any DSL for this?

Reply:
Not sure, I'm not editing it manually. I'm using the built-in UI Swing editor.

Reply:
It is much easier to review something DSL-like. [IJ has something](https://www.jetbrains.org/intellij/sdk/docs/tutorials/kotlin.html#ui-in-kotlin).

Reply:
I'll work on that on a separate PR :)

Reply:
Sure thing!